tools/distpartitioning/dataset_utils.py

```diff
@@ -528,16 +527,24 @@ def get_dataset(input_dir, graph_name, rank, world_size, num_parts, schema_map):
                 edge_file = os.path.join(input_dir, edge_file)
             logging.info(f'Loading edges of etype[{etype_name}] from {edge_file}')
 
-            read_options=pyarrow.csv.ReadOptions(use_threads=True, block_size=4096, autogenerate_column_names=True)
-            parse_options=pyarrow.csv.ParseOptions(delimiter=' ')
-            with pyarrow.csv.open_csv(edge_file, read_options=read_options, parse_options=parse_options) as reader:
-                for next_chunk in reader:
-                    if next_chunk is None:
-                        break
-
-                    next_table = pyarrow.Table.from_batches([next_chunk])
-                    src_ids.append(next_table['f0'].to_numpy())
-                    dst_ids.append(next_table['f1'].to_numpy())
+            if etype_info[constants.STR_FORMAT][constants.STR_NAME] == constants.STR_CSV:
+                read_options=pyarrow.csv.ReadOptions(use_threads=True, block_size=4096, autogenerate_column_names=True)
+                parse_options=pyarrow.csv.ParseOptions(delimiter=' ')
+                with pyarrow.csv.open_csv(edge_file, read_options=read_options, parse_options=parse_options) as reader:
+                    for next_chunk in reader:
+                        if next_chunk is None:
+                            break
+
+                        next_table = pyarrow.Table.from_batches([next_chunk])
+                        src_ids.append(next_table['f0'].to_numpy())
+                        dst_ids.append(next_table['f1'].to_numpy())
+            elif etype_info[constants.STR_FORMAT][constants.STR_NAME] == constants.STR_PARQUET:
```

Comment:
we'd better add test to cover this scenario. see more details in `//tests/tools/test_dist_part.py` and `//tests/tools/utils.py`.

Reply:
https://github.com/dmlc/dgl/blob/00f2099934ca78a3c156c4222edcc3245cee4e1c/tests/tools/utils.py#L134-L151